apps/vscode-extension/src/commands/generate/module.command.ts

```diff
@@ -13,18 +13,17 @@ import {
   stringifyOptions
 } from '../helpers';
 
+const getCurrentFolder = () => {
```

Comment:
You can use the method with the same name from `apps/vscode-extension/src/commands/helpers.ts`.
It has the exact same implementation.